src/main/drivers/serial_uart.c

```diff
@@ -385,4 +389,10 @@ UART_IRQHandler(UART, 7, 7)  // UART7 Rx/Tx IRQ Handler
 UART_IRQHandler(UART, 8, 8)  // UART8 Rx/Tx IRQ Handler
 #endif
 
+#if defined(STM32H7)
```

Comment:
Would it make sense to treat LPUART1 the same for G4 and H7, and have it mapped to 'UART 9' for both architectures?

Reply:
No strong though about numbering; just imagined that it would be better for users to keep possible uart numbers close so that they don't wonder where rest of the uarts are. No significant change code-wise.

It's going to be a change in `serial_uart_stm32g4xx.c` anyways.

Reply:
The reality is that most flight controllers do not support all possible UARTs, and the supported UARTs are often not in a contiguos range. So LPUART being 'UART 9' on all architectures will not be out of line with what we already have for users - in a way SOFTSERIAL does the same.
And I think on the code side having LPUART always on 9 will look a bit cleaner.

Reply:
Okay, UART9 on LPUART1 for G4 is ready at #9528. After the mod, this PR would just become a mere addition of `uartHardware` element for `UARTDEV_9`.